src/main/java/com/plivo/api/models/base/Meta.java

```diff
@@ -24,7 +26,8 @@ public String getPrevious() {
     return previous;
   }
 
-  public Long getTotalCount() {
+  public Long getTotalCount() throws NoSuchMethodException {
```

Comment:
On second thought, we're calling this out in release notes that the method is removed. What's your thought on actually removing this method instead?

cc @neel-plivo

Reply:
Just realised that this is the method of the base class.

Reply:
The other teams like voice, powerpack, etc might still be using the totalCount field so the method can't be removed from the base class. As for the child class, we couldn't find any provision for removing the method. The closest thing was to throw `NoSucMethodException`